test/parallel/test-fs-write.js

```diff
@@ -73,3 +73,13 @@ fs.open(fn2, args, 0o644, common.mustCall((err, fd) => {
   fs.write(fd, '', 0, 'utf8', written);
   fs.write(fd, expected, 0, 'utf8', done);
 }));
+
+fs.open(fn, 'w', 0o644, common.mustCall(function(err, fd) {
```

Comment:
I'd use something other than `fn` to prevent any issues with the tests interacting with one another.

Reply:
Done, thanks.